projects/1.18/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -1 +1,555 @@
+  "item.electrodynamics.upgradeimprovedsolarcell" : "改良太阳能电池升级",
+  "item.electrodynamics.upgradestator": "定子升级",
+  "item.electrodynamics.upgraderange" : "范围升级",
+  "item.electrodynamics.upgradeexperience" : "经验升级",
+  "item.electrodynamics.upgradeitemvoid" : "虚空升级",
+  "item.electrodynamics.upgradefortune" : "时运升级",
+  "item.electrodynamics.upgradesilktouch" : "精准采集升级",
+  "item.electrodynamics.upgradeunbreaking" : "耐久升级",
+  "item.electrodynamics.oxidevanadium" : "五氧化二钒",
+  "item.electrodynamics.oxidedisulfur" : "二氧化硫结晶",
+  "item.electrodynamics.oxidetrisulfur" : "三氧化硫",
+  "item.electrodynamics.oxidesulfurdichloride" : "二氯化硫结晶",
+  "item.electrodynamics.oxidethionylchloride" : "亚硫酰氯结晶",
+  "item.electrodynamics.oxidedititanium" : "二氧化钛",
+  "item.electrodynamics.oxidecalciumcarbonate" : "碳酸钙",
+  "item.electrodynamics.oxidesodiumcarbonate" : "碳酸钠",
+  "item.electrodynamics.oxidetitanium" : "氧化钛",
+  "item.electrodynamics.oxidechromite" : "氧化铬",
+  "item.electrodynamics.oxidechromiumdisilicide" : "二硅化铬",
+  "item.electrodynamics.crystalpotassiumchloride" : "氯化钾晶体",
+  "item.electrodynamics.insulation" : "绝缘材料",
+  "item.electrodynamics.insulationceramic" : "陶瓷绝缘材料",
+  "item.electrodynamics.motor" : "电动机",
+  "item.electrodynamics.battery" : "电池",
+  "item.electrodynamics.lithiumbattery" : "锂电池",
+  "item.electrodynamics.coil" : "线圈",
+  "item.electrodynamics.slag" : "金属矿渣",
+  "item.electrodynamics.molybdenumfertilizer" : "肥料",
+  "item.electrodynamics.titaniumheatcoil" : "钛加热元件",
+  "item.electrodynamics.coalcoke" : "焦炭",
+  "item.electrodynamics.multimeter" : "手持万用表",
+  "item.electrodynamics.solarpanelplate" : "太阳能板",
+  "item.electrodynamics.wrench" : "扳手",
+  "item.electrodynamics.drillheadsteel" : "钢钻头",
+  "item.electrodynamics.drillheadstainlesssteel" : "不锈钢钻头",
+  "item.electrodynamics.drillheadhslasteel" : "高强度低合金钢钻头",
+  "item.electrodynamics.drillheadtitanium" : "钛钻头",
+  "item.electrodynamics.drillheadtitaniumcarbide" : "碳化钛钻头",
+  "item.electrodynamics.guidebook" : "《电动力学指南》",
+  "item.electrodynamics.ceramicwet" : "陶瓷坯体",
+  "item.electrodynamics.ceramiccooked" : "陶瓷",
+  "item.electrodynamics.ceramicplate" : "陶瓷板",
+  "item.electrodynamics.ceramicfuse" : "陶瓷管保险丝",
+  "item.electrodynamics.concretemix" : "预拌混凝土",
+  "item.electrodynamics.rubberboots" : "橡胶靴",
+  "item.electrodynamics.electricdrill" : "电钻",
+  "item.electrodynamics.electricchainsaw" : "电锯",
+  "item.electrodynamics.electricbaton" : "电棍",
+  "item.electrodynamics.railgunkinetic" : "动能轨道炮",
+  "item.electrodynamics.railgunplasma" : "等离子轨道炮",
+  "item.electrodynamics.seismicscanner" : "地质扫描仪",
+  "item.electrodynamics.mechanizedcrossbow" : "机械弩",
+  "item.electrodynamics.compositearmorhelmet" : "复合装甲头盔",
+  "item.electrodynamics.compositearmorchestplate" : "复合装甲胸甲",
+  "item.electrodynamics.compositearmorleggings" : "复合装甲护腿",
+  "item.electrodynamics.compositearmorboots" : "复合装甲靴子",
+  "item.electrodynamics.nightvisiongoggles" : "夜视护目镜",
```

```suggestion
  "item.electrodynamics.nightvisiongoggles" : "夜视镜",
```
事实上“夜视镜”已经包涵了护目镜的功能，因此应该没必要多加护目
此外夜视镜的翻译就是night vision goggles

---

这个我觉得可改可不改